client/web/admin/src/views/System/Actionlog/Index.vue

```diff
@@ -98,6 +98,7 @@
         hover
         class="mb-0 small"
         head-variant="light"
+        :busy="isBusy"
```

Comment:
Just to keep consistent, can we call isBusy processing instead.
Also, does the spinner look like other resource lists? If not lets try to make it as similar as possible